inst/qml/MixedModelsBGLMM.qml

```diff
@@ -81,7 +81,7 @@ Form {
 			values:
 			[
 				{ label: qsTr("Binomial"),				value: "binomial"},
-				{ label: qsTr("Binomial (aggregated)"),	value: "binomial_agg"},
+				{ label: qsTr("Binomial (aggregated)"),	value: "binomialAgg"},
```

Comment:
If I am not mistaken, this option was already in the last release. You should add the value name change to `Upgrades.qml` to ensure backward compatibility (see https://github.com/jasp-stats/jasp-desktop/blob/development/Docs/development/jasp-upgrade-qml.md). The same concerns `MixedModelsGLMM.qml` (same option), and the `bootstrap_samples` -> `bootstrapSamples` in `MixedModelsOptions.qml`.

Reply:
thanks, didn't know about this